WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/Cells/RefundProductsTotalTableViewCell.swift

```diff
@@ -47,9 +47,9 @@ private extension RefundProductsTotalTableViewCell {
         taxTitleLabel.applyBodyStyle()
         taxPriceLabel.applyBodyStyle()
         productsRefundTitleLabel.applyBodyStyle()
-        productsRefundTitleLabel.font = .font(forStyle: .body, weight: .bold)
+        productsRefundTitleLabel.font = .font(forStyle: .body, weight: .semibold)
```

Comment:
⚠️ It's probably best if we stick to what's available in the styles. What about `applyHeadlineStyle()`? If that's not possible, we can make a different style in that file. Or maybe it's the headline style that should be changed to semi-bold. It looks like the design standard mentions that it should be semi-bold: https://woomobiledesignhandbook.wordpress.com/typography/

Reply:
Thanks for bringing this up! I totally missed that the `headline` style already defaults to semibold 🤦 
<img width="748" alt="Screen Shot 2020-10-06 at 3 18 12 PM" src="https://user-images.githubusercontent.com/562080/95256089-f5f13e80-07e7-11eb-9c8d-1333bc879312.png">

Reply:
Oh, cool! Where is this reference? 😄

Reply:
https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/typography/